Gruntfile.js

```diff
@@ -41,12 +49,7 @@ const applyPortShift = (env) => {
 
 Object.assign(process.env, applyPortShift(process.env))
 
-const chalk = require('chalk')
-const crypto = require('crypto')
-const { exec, fork } = require('child_process')
-const { copyFile, readFile } = require('fs/promises')
-const path = require('path')
-const { resolve } = path
+process.env.GI_VERSION = `${version}@${new Date().toISOString()}`
```

Comment:
What's the reasoning for having this date thing tacked on at the end?

Reply:
Its purpose is to ensure every build gets a distinct version, so that we don't have to bump our `package.json` version everytime a change should be deployed for live testing. Maybe I was wrong and it's actually not an issue.

Sure it will no longer be useful once we fully commit to using semver, unless maybe we keep doing private deployments for testing.

Also another suffix format for this thing might be nicer, if we keep one. Maybe a Unix timestamp rounded to the nearest second - just a suggestion.

Reply:
Hmm, ok, yeah we can keep this. Resolving